### ISSUE TITLE
Video to open in full screen

### ISSUE THREAD

**Issue by supportreq** [ladrua/Open-ownCloud#2]
The Video is not opening in horizontal full screen..

**Comment by ladrua** [comment]
Hey, thank you for testing! I will fix this after Im finished integrating sharing functionality. Same problem when previewing pictures.

**Comment by supportreq** [comment]
:+1:

**Comment by supportreq** [comment]
Hey @ladrua are you still working on this project?

**Comment by ladrua** [comment]
Yes, we’re just a bit short handed attm. Cant work on it until next week Im afraid.
